nomos-da/kzgrs/src/kzg.rs

```diff
@@ -76,6 +76,10 @@ mod test {
     use once_cell::sync::Lazy;
     use rand::{thread_rng, Fill};
 
+    // Import rayon only if parallel feature is enabled
+    #[cfg(feature = "parallel")]
+    use rayon::prelude::*;
+
```

Comment:
As I already said this is not the way to use this. What is the meaning of using rayon here?